diff -rU1 -I[$]OpenBSD ./etc/changelist ../51/etc/changelist
--- ./etc/changelist	Wed Aug 17 11:50:29 2011
+++ ../51/etc/changelist	Thu Feb  9 15:14:21 2012
@@ -15,3 +15,2 @@
 /etc/bootparams
-/etc/ccd.conf
 /etc/changelist
diff -rU1 -I[$]OpenBSD ./etc/login.conf ../51/etc/login.conf
--- ./etc/login.conf	Wed Aug 17 11:50:29 2011
+++ ../51/etc/login.conf	Thu Feb  9 15:14:21 2012
@@ -17,2 +17,3 @@
 # radius	Use radius authentication
+# reject	Use rejected authentication
 # skey		Use S/Key authentication
@@ -21,3 +22,5 @@
 # snk		Digital Pathways SecureNet Key authentication
+# tis		TIS Firewall Toolkit authentication
 # token		Generic X9.9 token authentication
+# yubikey	YubiKey authentication
 #
@@ -40,3 +43,3 @@
 default:\
-	:path=/usr/bin /bin /usr/sbin /sbin /usr/X11R6/bin /usr/local/bin:\
+	:path=/usr/bin /bin /usr/sbin /sbin /usr/X11R6/bin /usr/local/bin /usr/local/sbin:\
 	:umask=022:\
diff -rU1 -I[$]OpenBSD ./etc/protocols ../51/etc/protocols
--- ./etc/protocols	Wed Aug 17 11:50:29 2011
+++ ../51/etc/protocols	Thu Feb  9 15:14:21 2012
@@ -51,4 +51,4 @@
 sdrp	42	SDRP		# Source Demand Routing Protocol
-sip-sr	43	SIP-SR		# SIP Source Route
-sip-frag 44	SIP-FRAG	# SIP Fragment
+ipv6-route 43	IPv6-Route	# Routing Header for IPv6
+ipv6-frag 44	IPv6-Frag	# Fragment Header for IPv6
 idrp	45	IDRP		# Inter-Domain Routing Protocol
diff -rU1 -I[$]OpenBSD ./etc/services ../51/etc/services
--- ./etc/services	Wed Aug 17 11:50:29 2011
+++ ../51/etc/services	Thu Feb  9 15:14:21 2012
@@ -244,2 +244,4 @@
 daap		3689/udp			# Digital Audio Access Protocol
+sieve		4190/tcp			# ManageSieve Protocol
+sieve		4190/udp			# ManageSieve Protocol
 krb524		4444/tcp			# Kerberos 5->4
