diff -rU1 -I[$]OpenBSD ./etc/changelist ../50/etc/changelist
--- ./etc/changelist	Wed Mar  2 08:51:25 2011
+++ ../50/etc/changelist	Wed Aug 17 11:50:29 2011
@@ -40,2 +40,3 @@
 /etc/hostapd.conf
++/etc/hostname.*
 /etc/hosts
@@ -103,3 +104,2 @@
 /etc/pkg.conf
-/etc/portal.conf
 /etc/printcap
@@ -125,3 +125,2 @@
 /etc/sasyncd.conf
-/etc/security
 /etc/sensorsd.conf
diff -rU1 -I[$]OpenBSD ./etc/login.conf ../50/etc/login.conf
--- ./etc/login.conf	Wed Mar  2 08:51:26 2011
+++ ../50/etc/login.conf	Wed Aug 17 11:50:29 2011
@@ -86 +86,8 @@
 	:tc=default:
+
+#
+# Override resource limits for certain daemons started by rc.d(8)
+#
+bgpd:\
+	:openfiles-cur=512:\
+	:tc=daemon:
diff -rU1 -I[$]OpenBSD ./etc/man.conf ../50/etc/man.conf
--- ./etc/man.conf	Wed Mar  2 08:51:25 2011
+++ ../50/etc/man.conf	Wed Aug 17 11:50:29 2011
@@ -33,3 +33,3 @@
 # default
-_default	/usr/{share,X11R6,local}/man/
+_default	/usr/{share,X11R6,local,ports/infrastructure}/man/
 
diff -rU1 -I[$]OpenBSD ./etc/newsyslog.conf ../50/etc/newsyslog.conf
--- ./etc/newsyslog.conf	Wed Mar  2 08:51:25 2011
+++ ../50/etc/newsyslog.conf	Wed Aug 17 11:50:29 2011
@@ -13,3 +13,3 @@
 /var/log/secure				600  7     *    168   Z
-/var/log/wtmp				644  7     *    $W6D4 ZB
+/var/log/wtmp				644  7     *    $W6D4 B
 /var/log/xferlog			640  7     250  *     Z
diff -rU1 -I[$]OpenBSD ./etc/sysctl.conf ../50/etc/sysctl.conf
--- ./etc/sysctl.conf	Wed Mar  2 08:51:26 2011
+++ ../50/etc/sysctl.conf	Wed Aug 17 11:50:29 2011
@@ -10,3 +10,3 @@
 #net.inet.icmp.rediraccept=1	# 1=Accept ICMP redirects
-#net.inet6.icmp6.rediraccept=0	# 0=Don't accept IPv6 ICMP redirects
+#net.inet6.icmp6.rediraccept=1	# 1=Accept IPv6 ICMP redirects (for hosts)
 #net.inet6.ip6.forwarding=1	# 1=Permit forwarding (routing) of IPv6 packets
@@ -37,2 +37,4 @@
 #kern.watchdog.auto=0		# 0=Disable automatic watchdog(4) retriggering
+#kern.pool_debug=0		# 0=Disable pool corruption checks (faster)
+#hw.allowpowerdown=0		# 0=Disable power button shutdown
 #machdep.allowaperture=2	# See xf86(4)
@@ -44,4 +46,2 @@
 #kern.emul.aout=1		# enable running dynamic OpenBSD a.out bins
-#kern.emul.freebsd=1		# enable running FreeBSD binaries
 #kern.emul.linux=1		# enable running Linux binaries
-#kern.emul.svr4=1		# enable running SVR4 binaries
