--- etc/changelist	Sat Sep 10 17:15:06 2005
+++ 39/etc/changelist	Wed Feb 15 19:45:57 2006
@@ -38,2 +38,3 @@
 /etc/group
+/etc/hostapd.conf
 /etc/hosts
@@ -45,2 +46,3 @@
 /etc/inetd.conf
+/etc/ipsec.conf
 +/etc/isakmpd/private/local.key
--- etc/inetd.conf	Sat Sep 10 17:15:07 2005
+++ 39/etc/inetd.conf	Wed Feb 15 19:45:57 2006
@@ -8,3 +8,2 @@
 #ftp		stream	tcp6	nowait	root	/usr/libexec/ftpd	ftpd -US
-#127.0.0.1:8021	stream	tcp	nowait	root	/usr/libexec/ftp-proxy	ftp-proxy
 #shell		stream	tcp	nowait	root	/usr/libexec/rshd	rshd -L
@@ -45,7 +44,2 @@
 #time		dgram	udp6	wait	root	internal
-# Kerberos authenticated services
-#kshell		stream	tcp	nowait	root	/usr/libexec/rshd	rshd -k
-#ekshell	stream	tcp	nowait	root	/usr/libexec/rshd	rshd -Lk
-#ekshell2	stream	tcp	nowait	root	/usr/libexec/rshd	rshd -Lk
-#kauth		stream	tcp	nowait	root	/usr/libexec/kauthd	kauthd
 # RPC based services
--- etc/lynx.cfg	Sat Sep 10 17:15:28 2005
+++ 39/etc/lynx.cfg	Wed Feb 15 19:46:16 2006
@@ -5,6 +5,6 @@
 # $Format: "#PRCS LYNX_VERSION \"$ProjectVersion$\""$
-#PRCS LYNX_VERSION "2.8.5rel.2"
+#PRCS LYNX_VERSION "2.8.5rel.4"
 #
 # $Format: "#PRCS LYNX_DATE \"$ProjectDate$\""$
-#PRCS LYNX_DATE "Thu, 22 Apr 2004 16:08:10 -0700"
+#PRCS LYNX_DATE "Tue, 25 Oct 2005 17:40:26 -0700"
 #
--- etc/rc.conf	Sat Sep 10 17:15:07 2005
+++ 39/etc/rc.conf	Wed Feb 15 19:45:58 2006
@@ -33,2 +33,4 @@
 hotplugd_flags=NO	# for normal use: ""
+watchdogd_flags=NO	# for normal use: ""
+ftpproxy_flags=NO	# for normal use: ""
 
@@ -52,3 +54,3 @@
 
-# For enabling console mouse support (i386 and alpha only)
+# For enabling console mouse support (i386 alpha amd64)
 wsmoused_flags=NO	# for ps/2 or usb mice: "", serial: "-p /dev/cua00"
--- etc/ssh/ssh_config	Sat Sep 10 17:15:10 2005
+++ 39/etc/ssh/ssh_config	Wed Feb 15 19:46:01 2006
@@ -39 +39,4 @@
 #   EscapeChar ~
+#   Tunnel no
+#   TunnelDevice any:any
+#   PermitLocalCommand no
--- etc/ssh/sshd_config	Sat Sep 10 17:15:10 2005
+++ 39/etc/ssh/sshd_config	Wed Feb 15 19:46:01 2006
@@ -86,2 +86,3 @@
 #MaxStartups 10
+#PermitTunnel no
 
--- etc/sysctl.conf	Sat Sep 10 17:15:07 2005
+++ 39/etc/sysctl.conf	Wed Feb 15 19:45:59 2006
@@ -6,7 +6,8 @@
 #
-#net.inet.ip.forwarding=1	# 1=Permit forwarding (routing) of packets
-#net.inet6.ip6.forwarding=1	# 1=Permit forwarding (routing) of packets
+#net.inet.ip.forwarding=1	# 1=Permit forwarding (routing) of IPv4 packets
+#net.inet.ip.mforwarding=1	# 1=Permit forwarding (routing) of IPv4 multicast packets
+#net.inet6.ip6.forwarding=1	# 1=Permit forwarding (routing) of IPv6 packets
 #net.inet6.ip6.accept_rtadv=1	# 1=Permit IPv6 autoconf (forwarding must be 0)
-#net.inet.tcp.rfc1323=0		# 0=disable TCP RFC1323 extensions (for if tcp is slow)
-#net.inet.tcp.rfc3390=1		# 1=Enable RFC3390 for TCP window increasing
+#net.inet.tcp.rfc1323=0		# 0=Disable TCP RFC1323 extensions (for if tcp is slow)
+#net.inet.tcp.rfc3390=0		# 0=Disable RFC3390 for TCP window increasing
 #net.inet.esp.enable=0		# 0=Disable the ESP IPsec protocol
@@ -23,3 +24,3 @@
 #net.inet.ip.mtudisc=0		# 0=disable tcp mtu discovery
-#kern.usercrypto=1		# 1=enable userland use of /dev/crypto
+#kern.usercrypto=0		# 0=disable userland use of /dev/crypto
 #kern.splassert=2		# 2=enable with verbose error messages
