--- etc/changelist	Thu Mar  2 04:04:17 2006
+++ ../40/etc/changelist	Fri Sep  1 13:40:45 2006
@@ -29,2 +29,3 @@
 /etc/distfile
+/etc/dvmrpd.conf
 /etc/ethers
--- etc/ftpusers	Thu Mar  2 04:04:17 2006
+++ ../40/etc/ftpusers	Fri Sep  1 13:40:45 2006
@@ -40 +40,2 @@
 _hostapd
+_dvmrpd
--- etc/rc.conf	Thu Mar  2 04:04:18 2006
+++ ../40/etc/rc.conf	Fri Sep  1 13:40:45 2006
@@ -8,2 +8,3 @@
 			# be sure to enable multicast_router below.
+dvmrpd_flags=NO		# for normal use: ""
 ospfd_flags=NO		# for normal use: ""
@@ -19,2 +20,3 @@
 isakmpd_flags=NO	# for normal use: ""
+sasyncd_flags=NO	# for normal use: ""
 mopd_flags=NO		# for normal use: "-a"
@@ -23,2 +25,3 @@
 dhcpd_flags=NO		# for normal use: ""
+dhcrelay_flags=NO	# for normal use: "-i interface [server]"
 rtadvd_flags=NO		# for normal use: list of interfaces
@@ -35,2 +38,3 @@
 ftpproxy_flags=NO	# for normal use: ""
+hostapd_flags=NO	# for normal use: ""
 
@@ -63,2 +67,3 @@
 pf=NO			# Packet filter / NAT
+ipsec=NO		# IPsec
 portmap=NO		# Note: inetd(8) rpc services need portmap too
@@ -86,2 +91,3 @@
 pf_rules=/etc/pf.conf		# Packet filter rules file
+ipsec_rules=/etc/ipsec.conf	# IPsec rules file
 pflogd_flags=			# add more flags, ie. "-s 256"
--- etc/mail/aliases	Thu Mar  2 04:04:33 2006
+++ ../40/etc/mail/aliases	Fri Sep  1 13:41:01 2006
@@ -26,2 +26,3 @@
 _dhcp: /dev/null
+_dvmrpd: /dev/null
 _fingerd: /dev/null
--- etc/ssh/ssh_config	Thu Mar  2 04:04:20 2006
+++ ../40/etc/ssh/ssh_config	Fri Sep  1 13:40:48 2006
@@ -26,2 +26,4 @@
 #   HostbasedAuthentication no
+#   GSSAPIAuthentication no
+#   GSSAPIDelegateCredentials no
 #   BatchMode no
--- etc/ssh/sshd_config	Thu Mar  2 04:04:20 2006
+++ ../40/etc/ssh/sshd_config	Fri Sep  1 13:40:48 2006
@@ -93 +93,7 @@
 Subsystem	sftp	/usr/libexec/sftp-server
+
+# Example of overriding settings on a per-user basis
+#Match User anoncvs
+#	X11Forwarding no
+#	AllowTcpForwarding no
+#	ForceCommand cvs server
