diff -rU2 -I[$]OpenBSD ./etc/changelist ../47/etc/changelist
--- ./etc/changelist	Thu Jul  9 23:00:38 2009
+++ ../47/etc/changelist	Wed Mar 17 22:31:13 2010
@@ -51,4 +51,5 @@
 /etc/isakmpd/local.pub
 /etc/ksh.kshrc
+/etc/ldpd.conf
 /etc/locate.rc
 /etc/login.conf
@@ -114,4 +115,5 @@
 /etc/rpc
 /etc/ripd.conf
+/etc/rtadvd.conf
 /etc/sasyncd.conf
 /etc/security
@@ -131,4 +133,5 @@
 /etc/ssh/ssh_host_rsa_key.pub
 /etc/ssh/sshd_config
+/etc/ssl/cert.pem
 /etc/sudoers
 /etc/suid_profile
diff -rU2 -I[$]OpenBSD ./etc/ftpusers ../47/etc/ftpusers
--- ./etc/ftpusers	Thu Jul  9 23:00:38 2009
+++ ../47/etc/ftpusers	Wed Mar 17 22:31:13 2010
@@ -49,2 +49,4 @@
 _smtpd
 _rwalld
+_nsd
+_ldpd
diff -rU2 -I[$]OpenBSD ./etc/group ../47/etc/group
--- ./etc/group	Thu Jul  9 23:00:38 2009
+++ ../47/etc/group	Wed Mar 17 22:31:13 2010
@@ -62,4 +62,6 @@
 _smtpd:*:95:
 _rwalld:*:96:
+_nsd:*:97:
+_ldpd:*:98:
 dialer:*:117:
 nogroup:*:32766:
diff -rU2 -I[$]OpenBSD ./etc/ksh.kshrc ../47/etc/ksh.kshrc
--- ./etc/ksh.kshrc	Thu Jul  9 23:00:38 2009
+++ ../47/etc/ksh.kshrc	Wed Mar 17 22:31:13 2010
@@ -99,5 +99,5 @@
 		alias istripe='ilabel "$USER@$HOST ($tty)"'
 
-		wftp () { ilabel "ftp $*"; "ftp" $*; eval istripe; }
+		wftp () { ilabel "ftp $*"; "ftp" "$@"; eval istripe; }
 		wcd () { \cd "$@" && eval stripe; }
 		wssh ()
diff -rU2 -I[$]OpenBSD ./etc/mail/aliases ../47/etc/mail/aliases
--- ./etc/mail/aliases	Thu Jul  9 23:00:56 2009
+++ ../47/etc/mail/aliases	Wed Mar 17 22:31:18 2010
@@ -34,5 +34,7 @@
 _kadmin: /dev/null
 _kdc: /dev/null
+_ldpd: /dev/null
 _mopd: /dev/null
+_nsd: /dev/null
 _ntp: /dev/null
 _ospfd: /dev/null
diff -rU2 -I[$]OpenBSD ./etc/mail/smtpd.conf ../47/etc/mail/smtpd.conf
--- ./etc/mail/smtpd.conf	Thu Jul  9 23:00:57 2009
+++ ../47/etc/mail/smtpd.conf	Wed Mar 17 22:31:19 2010
@@ -8,4 +8,4 @@
 map "aliases" { source db "/etc/mail/aliases.db" }
 
-accept for local deliver to mbox
+accept for local alias aliases deliver to mbox
 accept for all relay
diff -rU2 -I[$]OpenBSD ./etc/newsyslog.conf ../47/etc/newsyslog.conf
--- ./etc/newsyslog.conf	Thu Jul  9 23:00:38 2009
+++ ../47/etc/newsyslog.conf	Wed Mar 17 22:31:13 2010
@@ -15,3 +15,4 @@
 /var/log/xferlog			640  7     250  *     Z
 /var/log/ppp.log			640  7     250  *     Z
-/var/log/pflog				600  3     250  *     ZB /var/run/pflogd.pid
+/var/log/pflog				600  3     250  *     ZB "pkill -HUP -u root -U root -t - -x pflogd"
+
diff -rU2 -I[$]OpenBSD ./etc/protocols ../47/etc/protocols
--- ./etc/protocols	Thu Jul  9 23:00:38 2009
+++ ../47/etc/protocols	Wed Mar 17 22:31:13 2010
@@ -143,2 +143,3 @@
 pfsync	240	PFSYNC		# PF Synchronization
 reserved 255	Reserved	#
+divert	258	DIVERT		# Divert pseudo-protocol [non IANA]
diff -rU2 -I[$]OpenBSD ./etc/services ../47/etc/services
--- ./etc/services	Thu Jul  9 23:00:39 2009
+++ ../47/etc/services	Wed Mar 17 22:31:13 2010
@@ -216,4 +216,5 @@
 prospero-np	1525/udp
 sa-msg-port	1646/udp
+l2tp		1701/udp			# Layer 2 Tunneling Protocol
 pptp		1723/tcp			# Point-to-Point Tunneling Protocol
 pptp		1723/udp			# Point-to-Point Tunneling Protocol
@@ -236,4 +237,5 @@
 mysql		3306/tcp			# MySQL
 iapp		3517/tcp	802-11-iapp	# IEEE 802.11f IAPP
+iscsi		3260/tcp			# ISCSI
 iapp		3517/udp	802-11-iapp	# IEEE 802.11f IAPP
 daap		3689/tcp			# Digital Audio Access Protocol
diff -rU2 -I[$]OpenBSD ./etc/sudoers ../47/etc/sudoers
--- ./etc/sudoers	Thu Jul  9 23:00:58 2009
+++ ../47/etc/sudoers	Wed Mar 17 22:31:19 2010
@@ -17,8 +17,9 @@
 
 # Defaults specification
-Defaults env_keep +="DESTDIR EDITOR FETCH_CMD FLAVOR FTPMODE GROUP MAKE"
-Defaults env_keep +="MULTI_PACKAGES NOMAN OKAY_FILES OWNER PKG_CACHE PKG_DBDIR"
-Defaults env_keep +="PKG_DESTDIR PKG_PATH PKG_TMPDIR PORTSDIR RELEASEDIR"
-Defaults env_keep +="SHARED_ONLY SSH_AUTH_SOCK SUBPACKAGE VISUAL WRKOBJDIR"
+Defaults env_keep +="DESTDIR DISTDIR EDITOR FETCH_CMD FLAVOR FTPMODE GROUP MAKE"
+Defaults env_keep +="MAKECONF MULTI_PACKAGES NOMAN OKAY_FILES OWNER PKG_CACHE"
+Defaults env_keep +="PKG_DBDIR PKG_DESTDIR PKG_PATH PKG_TMPDIR PORTSDIR"
+Defaults env_keep +="RELEASEDIR SHARED_ONLY SSH_AUTH_SOCK SUBPACKAGE VISUAL"
+Defaults env_keep +="WRKOBJDIR"
 
 # Uncomment to disable the lecture the first time you run sudo
diff -rU2 -I[$]OpenBSD ./etc/sysctl.conf ../47/etc/sysctl.conf
--- ./etc/sysctl.conf	Thu Jul  9 23:00:40 2009
+++ ../47/etc/sysctl.conf	Wed Mar 17 22:31:13 2010
@@ -36,5 +36,4 @@
 #kern.watchdog.auto=0		# 0=Disable automatic watchdog(4) retriggering
 #machdep.allowaperture=2	# See xf86(4)
-#machdep.apmwarn=10		# battery % when apm status messages enabled
 #machdep.apmhalt=1		# 1=powerdown hack, try if halt -p doesn't work
 #machdep.kbdreset=1		# permit console CTRL-ALT-DEL to do a nice halt
diff -rU2 -I[$]OpenBSD ./var/named/etc/named-dual.conf ../47/var/named/etc/named-dual.conf
--- ./var/named/etc/named-dual.conf	Thu Jul  9 23:00:41 2009
+++ ../47/var/named/etc/named-dual.conf	Wed Mar 17 22:31:14 2010
@@ -51,5 +51,5 @@
 	};
 
-	zone "0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa" {
+	zone "1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa" {
 		type master;
 		file "standard/loopback6.arpa";
diff -rU2 -I[$]OpenBSD ./var/named/etc/named-simple.conf ../47/var/named/etc/named-simple.conf
--- ./var/named/etc/named-simple.conf	Thu Jul  9 23:00:41 2009
+++ ../47/var/named/etc/named-simple.conf	Wed Mar 17 22:31:14 2010
@@ -49,5 +49,5 @@
 };
 
-zone "0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa" {
+zone "1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa" {
 	type master;
 	file "standard/loopback6.arpa";
diff -rU2 -I[$]OpenBSD ./var/named/etc/named.conf ../47/var/named/etc/named.conf
--- ./var/named/etc/named.conf	Thu Jul  9 23:00:41 2009
+++ ../47/var/named/etc/named.conf	Wed Mar 17 22:31:14 2010
@@ -49,5 +49,5 @@
 };
 
-zone "0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa" {
+zone "1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa" {
 	type master;
 	file "standard/loopback6.arpa";
diff -rU2 -I[$]OpenBSD ./var/named/standard/loopback6.arpa ../47/var/named/standard/loopback6.arpa
--- ./var/named/standard/loopback6.arpa	Thu Jul  9 23:00:41 2009
+++ ../47/var/named/standard/loopback6.arpa	Wed Mar 17 22:31:14 2010
@@ -1,5 +1,5 @@
 ; $OpenBSD: upgrade47.patch,v 1.1 2010/04/18 17:09:25 nick Exp $
 
-$ORIGIN 0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa.
+$ORIGIN 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa.
 $TTL 6h
 
@@ -12,3 +12,3 @@
 
 		NS	localhost.
-1		PTR	localhost.
+		PTR	localhost.
diff -rU2 -I[$]OpenBSD ./var/www/conf/mime.types ../47/var/www/conf/mime.types
--- ./var/www/conf/mime.types	Thu Jul  9 23:00:48 2009
+++ ../47/var/www/conf/mime.types	Wed Mar 17 22:31:15 2010
@@ -54,5 +54,5 @@
 application/octet-stream	bin dms lha lzh exe class so dll
 application/oda			oda
-application/ogg			ogg
+application/ogg			ogx oggx
 application/parityfec
 application/pdf			pdf
@@ -268,4 +268,21 @@
 application/vnd.novadigm.edx
 application/vnd.novadigm.ext
+application/vnd.oasis.opendocument.chart			odc
+application/vnd.oasis.opendocument.chart-template		otc
+application/vnd.oasis.opendocument.database			odb
+application/vnd.oasis.opendocument.formula			odf
+application/vnd.oasis.opendocument.formula-template		otf
+application/vnd.oasis.opendocument.graphics			odg
+application/vnd.oasis.opendocument.graphics-template		otg
+application/vnd.oasis.opendocument.image			odi
+application/vnd.oasis.opendocument.image-template		oti
+application/vnd.oasis.opendocument.presentation			odp
+application/vnd.oasis.opendocument.presentation-template	otp
+application/vnd.oasis.opendocument.spreadsheet			ods
+application/vnd.oasis.opendocument.spreadsheet-template		ots
+application/vnd.oasis.opendocument.text				odt
+application/vnd.oasis.opendocument.text-master			odm
+application/vnd.oasis.opendocument.text-template		ott
+application/vnd.oasis.opendocument.text-web			oth
 application/vnd.obn
 application/vnd.osa.netdeploy
@@ -418,4 +435,5 @@
 audio/mp4a-latm
 audio/mpeg			mpga mp2 mp3
+audio/ogg			oga ogga spx
 audio/parityfec
 audio/pcma
@@ -581,4 +599,5 @@
 video/mpeg			mpeg mpg mpe
 video/nv
+video/ogg			ogv oggv
 video/parityfec
 video/pointer
