--- ../44/etc/changelist	Tue Aug 12 18:51:04 2008
+++ etc/changelist	Sat Feb 28 16:29:02 2009
@@ -153,4 +153,8 @@
 /var/named/etc/named.conf
 +/var/named/etc/rndc.key
+/var/named/etc/root.hint
+/var/named/standard/localhost
+/var/named/standard/loopback
+/var/named/standard/loopback6.arpa
 /var/www/conf/access.conf
 /var/www/conf/httpd.conf
--- ../44/etc/ftpusers	Tue Aug 12 18:51:04 2008
+++ etc/ftpusers	Sat Feb 28 16:29:02 2009
@@ -46,2 +46,3 @@
 _rtadvd
 _ypldap
+_btd
--- ../44/etc/ksh.kshrc	Tue Aug 12 18:51:04 2008
+++ etc/ksh.kshrc	Sat Feb 28 16:29:02 2009
@@ -142,5 +142,4 @@
 	alias j=jobs
 	alias o='fg %-'
-	alias ls='ls -gCF'
 
 # add your favourite aliases here
--- ../44/etc/lynx.cfg	Tue Aug 12 18:51:21 2008
+++ etc/lynx.cfg	Sat Feb 28 16:29:14 2009
@@ -364,4 +364,5 @@
 #    Latin 3 (ISO-8859-3)               iso-8859-3
 #    Latin 4 (ISO-8859-4)               iso-8859-4
+#    Baltic Rim (ISO-8859-13)           iso-8859-13
 #    Baltic Rim (cp775)                 cp775
 #    Baltic Rim (windows-1257)          windows-1257
--- ../44/etc/mail/aliases	Tue Aug 12 18:51:21 2008
+++ etc/mail/aliases	Sat Feb 28 16:29:14 2009
@@ -24,4 +24,5 @@
 _afs: /dev/null
 _bgpd: /dev/null
+_btd: /dev/null
 _dhcp: /dev/null
 _dvmrpd: /dev/null
--- ../44/etc/mail/trusted-users	Tue Aug 12 18:51:22 2008
+++ etc/mail/trusted-users	Sat Feb 28 16:29:15 2009
@@ -2,5 +2,5 @@
 #
 # This file contains a list of users (one per line) that may set their
-# envelope # "from" address to a different user via "sendmail -f".
+# envelope "from" address to a different user via "sendmail -f".
 # Common values include "majordomo", "mailman" and "www".
 # In other words, if a daemon (or mailing list manager) sends mail purporting
--- ../44/etc/shells	Tue Aug 12 18:51:05 2008
+++ etc/shells	Sat Feb 28 16:29:02 2009
@@ -1,6 +1,7 @@
-#	$OpenBSD: upgrade45.patch,v 1.1 2009/03/22 18:32:37 nick Exp $
-# List of acceptable shells for chpass(1).
-# Ftpd will not allow users to connect who are not using
-# one of these shells.
+#	$OpenBSD: upgrade45.patch,v 1.1 2009/03/22 18:32:37 nick Exp $
+#
+# list of acceptable shells for chpass(1).
+# ftpd(8) will not allow users to connect who are not using
+# one of these shells, unless the user is listed in /etc/ftpchroot.
 /bin/sh
 /bin/csh
--- ../44/etc/ssh/ssh_config	Tue Aug 12 18:51:13 2008
+++ etc/ssh/ssh_config	Sat Feb 28 16:29:08 2009
@@ -38,5 +38,5 @@
 #   Protocol 2,1
 #   Cipher 3des
-#   Ciphers aes128-cbc,3des-cbc,blowfish-cbc,cast128-cbc,arcfour,aes192-cbc,aes256-cbc
+#   Ciphers aes128-ctr,aes192-ctr,aes256-ctr,arcfour256,arcfour128,aes128-cbc,3des-cbc
 #   MACs hmac-md5,hmac-sha1,umac-64@openssh.com,hmac-ripemd160
 #   EscapeChar ~
--- ../44/var/named/etc/named.conf	Tue Aug 12 18:51:07 2008
+++ var/named/etc/named.conf	Sat Feb 28 16:29:04 2009
@@ -53,12 +53,4 @@
 	file "standard/loopback6.arpa";
 	allow-transfer { localhost; };
-};
-
-zone "com" {
-	type delegation-only;
-};
-
-zone "net" {
-	type delegation-only;
 };
 
