diff -rU1 -I[$]OpenBSD ./etc/changelist ../48/./etc/changelist
--- ./etc/changelist	Wed Mar 17 22:31:13 2010
+++ ../48/./etc/changelist	Mon Aug 16 10:46:24 2010
@@ -47,2 +47,3 @@
 /etc/ifstated.conf
++/etc/iked.conf
 /etc/inetd.conf
@@ -98,3 +99,2 @@
 /etc/pf.os
-/etc/phones
 /etc/portal.conf
@@ -103,2 +103,3 @@
 /etc/protocols
++/etc/pwd.db
 /etc/rbootd.conf
@@ -126,2 +127,3 @@
 /etc/snmpd.conf
++/etc/spwd.db
 /etc/ssh/ssh_config
diff -rU1 -I[$]OpenBSD ./etc/ftpusers ../48/./etc/ftpusers
--- ./etc/ftpusers	Wed Mar 17 22:31:13 2010
+++ ../48/./etc/ftpusers	Mon Aug 16 10:46:24 2010
@@ -52 +52,4 @@
 _ldpd
+_sndio
+_ldapd
+_iked
diff -rU1 -I[$]OpenBSD ./etc/mail/aliases ../48/./etc/mail/aliases
--- ./etc/mail/aliases	Wed Mar 17 22:31:18 2010
+++ ../48/./etc/mail/aliases	Mon Aug 16 10:46:33 2010
@@ -32,2 +32,3 @@
 _identd: /dev/null
+_iked: /dev/null
 _isakmpd: /dev/null
@@ -35,2 +36,3 @@
 _kdc: /dev/null
+_ldapd: /dev/null
 _ldpd: /dev/null
@@ -52,2 +54,3 @@
 _smtpd: /dev/null
+_sndio: /dev/null
 _snmpd: /dev/null
diff -rU1 -I[$]OpenBSD ./etc/man.conf ../48/./etc/man.conf
--- ./etc/man.conf	Wed Mar 17 22:31:13 2010
+++ ../48/./etc/man.conf	Mon Aug 16 10:46:24 2010
@@ -18,13 +18,13 @@
 _build		.0.gz		/usr/bin/gzcat %s
-_build		.[1-9n]		/usr/bin/nroff -man %s
-_build		.[1-9n].Z	/usr/bin/zcat %s | /usr/bin/nroff -man
-_build		.[1-9n].gz	/usr/bin/gzcat %s | /usr/bin/nroff -man
-_build		.[1-9][a-z]	/usr/bin/nroff -man %s
-_build		.[1-9][a-z].Z	/usr/bin/zcat %s | /usr/bin/nroff -man
-_build		.[1-9][a-z].gz	/usr/bin/gzcat %s | /usr/bin/nroff -man
-_build		.tbl		/usr/bin/tbl %s | /usr/bin/nroff -man
-_build		.tbl.Z		/usr/bin/zcat %s | /usr/bin/tbl | /usr/bin/nroff -man
-_build		.tbl.gz		/usr/bin/gzcat %s | /usr/bin/tbl | /usr/bin/nroff -man
-_build		.me		/usr/bin/nroff -me %s 2>/dev/null | cat -s
-_build		.ms		/usr/bin/nroff -ms %s 2>/dev/null | cat -s
+_build		.[1-9n]		/usr/bin/nroff -Tascii -man %s
+_build		.[1-9n].Z	/usr/bin/zcat %s | /usr/bin/nroff -Tascii -man
+_build		.[1-9n].gz	/usr/bin/gzcat %s | /usr/bin/nroff -Tascii -man
+_build		.[1-9][a-z]	/usr/bin/nroff -Tascii -man %s
+_build		.[1-9][a-z].Z	/usr/bin/zcat %s | /usr/bin/nroff -Tascii -man
+_build		.[1-9][a-z].gz	/usr/bin/gzcat %s | /usr/bin/nroff -Tascii -man
+_build		.tbl		/usr/bin/tbl %s | /usr/bin/nroff -Tascii -man
+_build		.tbl.Z		/usr/bin/zcat %s | /usr/bin/tbl | /usr/bin/nroff -Tascii -man
+_build		.tbl.gz		/usr/bin/gzcat %s | /usr/bin/tbl | /usr/bin/nroff -Tascii -man
+_build		.me		/usr/bin/nroff -Tascii -me %s 2>/dev/null | cat -s
+_build		.ms		/usr/bin/nroff -Tascii -ms %s 2>/dev/null | cat -s
 
diff -rU1 -I[$]OpenBSD ./etc/newsyslog.conf ../48/./etc/newsyslog.conf
--- ./etc/newsyslog.conf	Wed Mar 17 22:31:13 2010
+++ ../48/./etc/newsyslog.conf	Mon Aug 16 10:46:24 2010
@@ -17,2 +17 @@
 /var/log/pflog				600  3     250  *     ZB "pkill -HUP -u root -U root -t - -x pflogd"
-
diff -rU1 -I[$]OpenBSD ./etc/remote ../48/./etc/remote
--- ./etc/remote	Wed Mar 17 22:31:13 2010
+++ ../48/./etc/remote	Mon Aug 16 10:46:25 2010
@@ -4,3 +4,3 @@
 # remote -- remote host description database
-# see tip(1), cgetcap(3), phones(5), remote(5)
+# see tip(1), cgetcap(3), remote(5)
 #
@@ -8,6 +8,4 @@
 #
-#   at	ACU type
 #   br	baud rate (defaults to 300)
 #   dc	direct connect
-#   du	make a call flag (dial up)
 #   dv	device to use for the tty
@@ -17,3 +15,2 @@
 #   pa	parity
-#   pn	phone #, '\@' means use the phones(5) file
 #   tc	include the named system description
@@ -52,5 +49,2 @@
 
-dialup:\
-	:du:at=hayes:pn=\@:
-
 doshost:\
@@ -61,5 +55,2 @@
 
-# Sample directly connected lines. Directly connected lines are
-# most commonly used for serial consoles.
-#
 tty00|For hp300,i386,mac68k,macppc,mvmeppc,vax:\
@@ -69,8 +60 @@
 	:dv=/dev/ttya:tc=direct:tc=unixhost:
-
-# Sample dial out lines.
-#
-cua00|For hp300,i386,mac68k,macppc,mvmeppc,vax:\
-	:dv=/dev/cua00:tc=dialup:tc=unixhost:
-cuaa|For sparc,mvme68k:\
-	:dv=/dev/cuaa:tc=dialup:tc=unixhost:
diff -rU1 -I[$]OpenBSD ./etc/services ../48/./etc/services
--- ./etc/services	Wed Mar 17 22:31:13 2010
+++ ../48/./etc/services	Mon Aug 16 10:46:25 2010
@@ -153,2 +153,4 @@
 ldaps		636/udp
+ldp		646/tcp
+ldp		646/udp
 rsync		873/tcp				# rsync server
diff -rU1 -I[$]OpenBSD ./etc/ssl/x509v3.cnf ../48/./etc/ssl/x509v3.cnf
--- ./etc/ssl/x509v3.cnf	Wed Mar 17 22:31:18 2010
+++ ../48/./etc/ssl/x509v3.cnf	Mon Aug 16 10:46:33 2010
@@ -3,2 +3,3 @@
 CERTUSAGE		= digitalSignature,keyCertSign,cRLSign
+EXTCERTUSAGE		= serverAuth,clientAuth
 CERTIP			= 0.0.0.0
@@ -20,2 +21,3 @@
 subjectAltName=IP:$ENV::CERTIP
+extendedKeyUsage=$ENV::EXTCERTUSAGE
 
@@ -26 +28,2 @@
 subjectAltName=DNS:$ENV::CERTFQDN
+extendedKeyUsage=$ENV::EXTCERTUSAGE
diff -rU1 -I[$]OpenBSD ./etc/sysctl.conf ../48/./etc/sysctl.conf
--- ./etc/sysctl.conf	Wed Mar 17 22:31:13 2010
+++ ../48/./etc/sysctl.conf	Mon Aug 16 10:46:25 2010
@@ -39,2 +39,3 @@
 #machdep.kbdreset=1		# permit console CTRL-ALT-DEL to do a nice halt
+#machdep.lidsuspend=1		# laptop lid closes cause a suspend
 #machdep.userldt=1		# allow userland programs to play with ldt,
@@ -42,5 +43,3 @@
 #kern.emul.aout=1		# enable running dynamic OpenBSD a.out bins
-#kern.emul.bsdos=1		# enable running BSD/OS binaries
 #kern.emul.freebsd=1		# enable running FreeBSD binaries
-#kern.emul.ibcs2=1		# enable running iBCS2 binaries
 #kern.emul.linux=1		# enable running Linux binaries
