Index: usr.bin/ssh/sshconnect.c
===================================================================
RCS file: /cvs/src/usr.bin/ssh/sshconnect.c,v
retrieving revision 1.55
diff -u -r1.55 sshconnect.c
--- usr.bin/ssh/sshconnect.c	2000/02/15 13:08:01	1.55
+++ usr.bin/ssh/sshconnect.c	2000/02/17 07:20:29
@@ -148,7 +148,9 @@
 	 */
 	if (privileged) {
 		int p = IPPORT_RESERVED - 1;
-		sock = rresvport_af(&p, family);
+		if (family != AF_INET)
+			return -1;
+		sock = rresvport(&p);
 		if (sock < 0)
 			error("rresvport: af=%d %.100s", family, strerror(errno));
 		else
